pyanaconda/modules/network/firewall/Makefile.am

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
You should add this file to `configure.ac`.

Reply:
Added.